pythermalcomfort/models/work_capacity.py

```diff
@@ -0,0 +1,243 @@
+from typing import Union
+
+import numpy as np
+
+from pythermalcomfort.classes_input import (
+    WorkCapacityHothapsInputs,
+    WorkCapacityStandardsInputs,
+)
+from pythermalcomfort.classes_return import WorkCapacity
+
+
+def workcapacity_dunne(wbgt: Union[float, list[float]], intensity: str) -> WorkCapacity:
+    """
+    Estimate work capacity due to heat based ISO standards as described by Dunne et al
```

Comment:
please add reference, example `Returns the value of the Cooling Effect (`CE`_) calculated in compliance
    with the ASHRAE 55 2020 Standard [ASHRAE_55_2023]_.`